hazelcast/connection.py

```diff
@@ -45,6 +45,11 @@ def __init__(self, initial_backoff, max_backoff, multiplier, cluster_connect_tim
         self._cluster_connect_attempt_begin = None
         self._current_backoff = None
 
+        if cluster_connect_timeout == sys.maxsize:
```

Comment:
`sys.maxsize` is an int, as far as I see we document `cluster_connect_timeout` as a float. Would using `float('inf')` work better?

Reply:
Changed it as you suggested